src/pages/signIn/SignScreen.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Text, View, SafeAreaView } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import styles from './style'
+import { t } from '../../localization/i18n'
+import AppBar from '../../components/appBar/AppBar'
+import LoginForm from './components/loginForm/LoginForm'
+import SignWithButton from './components/signWithButton/SignWithButton'
+import { AppColors, AppWords, Icons, LanguageLocalizationKey, PageName } from '../../constants/constants'
+
+class SignScreen extends React.Component {
+    render() {
+        const { navigation } = this.props
+        return (
+            <LinearGradient
+                colors={[AppColors.lightBlue, AppColors.darkBlue]}
+                start={{ x: 0, y: 0 }}
+                end={{ x: 1, y: 0 }}
+            >
+                <SafeAreaView>
+                    <View style={styles.background}>
+                        <AppBar
+                            text={t('title', LanguageLocalizationKey.signIn)}
+                            pageName={PageName.tabs}
+                            navigation={navigation}
+                        />
+                        <Body navigation={navigation}/>
+                    </View>
+                </SafeAreaView>
+            </LinearGradient>
+        )
+    }
+}
+class Body extends React.Component {
```

Comment:
Use the Body class content inside the function of SignScreen.
Ex. ` 
render content = () => {
    return (Body content)
}